### ISSUE TITLE
Websites don't open in assigned containers when opened from command line, unless a FF window is already open

### ISSUE THREAD

**Issue by scenicforest** [mozilla/multi-account-containers#1534]
- Multi-Account Containers Version: 6.1.1
- OS: Ubuntu 19.10
- Firefox Version: 69.0.3 (Build ID 20191010103742)
- Other installed Add-ons + Version + Enabled/Disabled-Status:
(Name, Version, Enabled, ID)
Amazon.com	1.1	true	amazondotcom@search.mozilla.org
Bing	1.0	true	bing@search.mozilla.org
DuckDuckGo	1.0	true	ddg@search.mozilla.org
eBay	1.0	true	ebay@search.mozilla.org
Firefox Multi-Account Containers	6.1.1	true	@testpilot-containers
Google	1.0	true	google@search.mozilla.org
HTTPS Everywhere	2019.6.27	true	https-everywhere@eff.org
Privacy Badger	2019.10.8	true	jid1-MnnxcxisBPnSXQ@jetpack
Twitter	1.0	true	twitter@search.mozilla.org
Wikipedia (en)	1.0	true	wikipedia@search.mozilla.org
User-Agent Switcher	0.2.6	false	{75afe46a-7a50-4c6b-b866-c43a1075b071}

### Actual behavior
This is what happens when I open one or more websites from the command line: unless an instance of Firefox is already open, at least one of the websites does not open in its assigned containers, even though "Always open in ___" box is checked. If trying to open multiple websites, then the behavior of which open in assigned container and which don't is unpredictable. 

### Expected behavior
Websites should open in assigned containers if "Always open in ___" box is checked.

### Steps to reproduce
1. Create one or more containers. 
2. Assign one or more websites to containers.
3. Check "Always open in ___" for each website assigned to a container. 
4. Open each website, again, in a new tab. 
5. In each tab, check "Remember my decision for this site"; click button that says "Open in ____ Container"
6. Close all Firefox windows.
7. Open the websites in a single command line request to Firefox. Example:
`firefox 'https://www.vidangel.com/browse' 'https://www.commonsensemedia.org/' 'https://www.imdb.com/'`

One or more of the websites will not open in their assigned container. 

If, however, you already have a Firefox window open when you run the command in Step 7, then it will work properly: all sites will open in their assigned containers.

**Comment by kendallcorner** [comment]
Hi. This looks like a duplicate of #365. Please vote or discuss on that issue. I'll close this, but feel free to reopen if you think it's a different issue.

**Comment by SuryanathanP** [comment]
I don't think #1534 and #365 are same issues:

#365 addresses the issue where you want to open a site specified with the --url parameter in a specific container by passing along a --container parameter.
#1534 addresses the issue where a site is already bound to a container, and when opened from the command line using only the --url parameter, it should be opened in the bound container.
